packages/namekit-react/src/components/PriceSymbol.tsx

```diff
@@ -0,0 +1,88 @@
+import { PriceCurrencyFormat, Currency } from "@namehash/ens-utils";
+import { Tooltip } from "@namehash/nameguard-react";
+
+import EthereumCircleIcon from "./EthereumCircleIcon";
+import { UsdcSymbol } from "./UsdcSymbol";
+import { WethSymbol } from "./WethSymbol";
+import { DaiSymbol } from "./DaiSymbol";
+import React from "react";
+
+export enum PriceDisplaySize {
+  Micro = "text-xs md:text-sm font-normal",
+  Small = "text-sm font-semibold",
+  Medium = "text-xl font-semibold",
+  Large = "text-xl font-semibold md:text-2xl md:font-bold",
+}
+
+export enum PriceSymbolPosition {
+  Left = "mr-1.5",
+  Right = "ml-1.5",
+}
+
+export enum PriceDisplayPosition {
+  Right = "flex inline-flex items-end space-x-2",
+  Bottom = "flex flex-col text-right items-end space-y-1",
+}
+
+export enum PriceSymbology {
+  Acronym = "Acronym",
+  Symbol = "Symbol",
+}
+
+interface PriceSymbolProps {
+  currency: Currency;
+  size: PriceDisplaySize;
+  symbolFillColor?: string;
```

Comment:
Could you please document what the purpose of this `symbolFillColor` prop is for? Are we actually using it somewhere? What goal does it help to achieve?

Reply:
In storybook there is one example of this usage: we use it for displaying the symbol in the same color as the text in alternative price when gray... Both examples of a gray ethereum icon and a blue ethereum icon, which represents the ability of changing the symbol color, can be found in storybook docs, please take a look and let me know your thoughts! 😃

Reply:
@FrancoAguzzi Based on the discussion you've been having with Jamie on ideas related to this, please let me know your advice on how we proceed.

Reply:
For now, no updates will be done to our handling of SVGs

I will do another effort around this, after the main DomainCard components are ready

What do you think > [Refine use of HTML in SVG icons](https://app.shortcut.com/ps-web3/story/25228/refine-use-of-html-in-svg-icons)?

Reply:
@FrancoAguzzi I'm ok to handle this in a separate story.